src/components/TutorialsOverview.vue

```diff
@@ -97,12 +96,12 @@ export default {
   },
   provide() {
     return {
-      references: this.references,
       store: this.store,
     };
   },
   created() {
     this.store.reset();
+    this.store.setReferences(this.references);
```

Comment:
In this file we aren't injecting a default for state for his.store.reset() or this.store.setReferences(). Could you add it?

Reply:
In this file we are directly importing the `TutorialsOverviewStore` from the source file, which I believe included the defaults.